app/src/main/java/com/gh4a/utils/ApiKeyHelper.java

```diff
@@ -0,0 +1,24 @@
+package com.gh4a.utils;
+
+import com.gh4a.BuildConfig;
+
+public class ApiKeyHelper {
+    private static final String GITHUB_FDROID_CLIENT_ID = "SOMETEXT";
+    private static final String GITHUB_FDROID_SECRET = "SOMETEXT";
```

Comment:
F-droid metadata is all open as well, all the other apps do it like that as well. I can get your keys from the apk in 5 minutes no matter how hard you obfuscate, so it's not a true thing to be concerned about :+1: